### ISSUE TITLE
How to connect to Paymium?

### ISSUE THREAD

**Issue by ojali** [ccxt/ccxt#1874]
Hello,

I'm trying to connect to PAYMIUM with the code below :
```
	let paymium = new ccxt.paymium ({
        'apiKey': 'here my public key',
        'secret': 'here mys secret key',
    })
```
I have a error 401 Unauthorized ...
With other exchanges like KRAKEN it works perfectly.
Is there another parameter to add to connect to this exchange? 
Who could help please ?

Thnak you for help :)

**Comment by kroitor** [comment]
Hi!

Is it a fresh new paymium key you're trying? Or did you use it previously with any other software?

**Comment by ojali** [comment]
I try with 2 keys, but these keys have never been used before.... So if you think about nonce problem, it is not the reason... :)

**Comment by kroitor** [comment]
@ojali sorry for a long wait... can you, please, post the smallest possible snippet of your code (but still, a complete one) to reproduce the problem (without your keys) ? I mean, the lines that you've copied don't tell me too much, need to see more code and verbose output, as described here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

Thx, standing by for more info from you...

**Comment by adam314315** [comment]
Did somebody finally succeeded to connect. On y side I am reporting the same 401 access authorization error while trying to use Paymium. (Yet I succeeded with Cryptopia exchange before). 

Here is my simple code using the Api Token and secret key generated via Paymium. Could you let me know any progress regarding this exchange ?

       $paymium = new paymium([
          'apiKey' => $this->paymiumPublicKey,
          'secret' => $this->paymiumPrivateKey,
        ]);
       echo '<pre>';print_r($paymium->fetch_balance ());echo '</pre>';

**Comment by kroitor** [comment]
@adam314315 the problem with Paymium is that it won't accept our signup due to territorial restrictions, so we can't get an account for debugging. Even their sandbox does not allow our numbers... If you could help us obtain a yet-unused new API key with zeroed balances there (like [other users do sometimes](https://github.com/ccxt/ccxt/issues/54)), we would be able to debug and test it then. This would speed it up significantly and the issue would then be solved in hours most likely.

**Comment by adam314315** [comment]
@kroitor Ok i can understand. Can you give me an email address to send you temporar access to a developer account ?

**Comment by kroitor** [comment]
@adam314315 my email is igor.kroitor@gmail.com, thx!

**Comment by kroitor** [comment]
@adam314315 got it, thank you, will report here as soon as I fix it, most likely will get to it in several hours.

**Comment by kroitor** [comment]
This issue has been fixed in 1.12.67. Tons of thanks to @adam314315 for the assistance. We will be happy if you report back whether it now work normally for you or not, should be ok as far as we tested on our side. Thx again!

**Comment by adam314315** [comment]
@kroitor Yes. Working now. Thanks for assistance.
